src/context/modal-context.jsx

```diff
@@ -57,11 +72,11 @@ const ModalProvider = ({ children }) => {
         <ConfirmDialog
           cancelButton='No'
           confirmButton='Yes'
-          message='Are you sure you want to close?'
+          message={dialogConfig.message}
           onConfirm={handleConfirmClose}
           onDismiss={() => setConfirmDialogOpen(false)}
           open={confirmDialogOpen}
-          title='Confirm Close'
+          title={dialogConfig.title}
```

Comment:
```suggestion
          title={t(dialogConfig.title)}
```

Reply:
Thank you for pointing on my mistakes, I changed the messages and translations